XCUITests/NavigationTest.swift

```diff
@@ -345,6 +345,41 @@ class NavigationTest: BaseTestCase {
         waitForValueContains(app.textFields["url"], value: website_2["moreLinkLongPressInfo"]!)
     }
 
+	func testLongPressOnAddressBar() {
+
+		//Navigate to a URL and perform a long press on the non-highlighted address bar
```

Comment:
Indentation does not look good...extra space here and no space in some lines below, could you please check that? Thanks!